server/config/utils_test.go

```diff
@@ -80,7 +80,9 @@ func TestDesanitize(t *testing.T) {
 }
 
 func TestFixInvalidLocales(t *testing.T) {
-	utils.TranslationsPreInit()
+	// utils.TranslationsPreInit errors when TestFixInvalidLocales is run as part of testing the package,
+	// but doesn't error when the test is run individually.
+	_ = utils.TranslationsPreInit()
```

Comment:
I haven't found out why, though.

Reply:
Do we know the error? Maybe we can create a ticket to track or create a HW issue.